e2e_samples/dataset_versioning/datafactory/config/README.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+This folder is for SQL query which creata following required tables and stored procedures.
+
+- `source` table which stores sample data
+- `WaterMark` table storing table name and its watermark value
+- `usp_UpdateWatermark` stored procedure to update watermark value
+
+## How to create required tables and stored procedure
+
+1. Install [Azure Data Studio](https://docs.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver15)
+1. Get user id and connection string for SQL Database
+   1. Sign-in to Azure and go to provisioned resource group.
+   1. Go to Azure KeyVault
+   1. Click [Access policies] in left pane, then click [+ Add Access Policy]
+   1. Click [Secret permissions] dropbox, then check [Select all] check box.
+   1. Click [None selected], then add yourself. Click [Add] to save setting.
```

Comment:
I think it would be helpful to say click [None selected] next to "Select principal"

Reply:
Have updated.